sjb/config/common/test_cases/origin_release_install_azure_39.yml

```diff
@@ -160,7 +160,7 @@ extensions:
           -e "openshift_azure_resource_group_name=$RESOURCE_GROUP" \
           -e "openshift_azure_resource_location=eastus" \
           -e "openshift_azure_input_image_ns=$RESOURCE_GROUP" \
-          -e "openshift_azure_input_image_prefix=centos7" \
+          -e "openshift_azure_input_image_name=centos7-origin" \
```

Comment:
AFAIK the current prefixes for the centos node images are `centos7-3.9-` and `centos7-3.10-` - are we changing them?

Reply:
this image is built only for testing, it's the only image in temporary `$RESOURCE_GROUP`, will be deleted at the end of the test; so this is just easy name which can be used for any version.

Reply:
ack